### PR TITLE
CA-26/Firebase Cloud Messaging and Analytics

### PR DESCRIPTION
**Pull request by rock3r**
We can now send push notifications to the app from the Firebase console. Not much more done for now. Firebase Analytics has been enabled too, since we had that in anyway.

### CODE REVIEW

File: app/src/main/java/com/connfa/push/PushMessagingService.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package com.connfa.push;
+
+import com.google.firebase.messaging.FirebaseMessagingService;
+import com.google.firebase.messaging.RemoteMessage;
+
+public class PushMessagingService extends FirebaseMessagingService {
+
+    @Override
+    public void onMessageReceived(RemoteMessage remoteMessage) {
```
Comment (fourlastor) [inline app/src/main/java/com/connfa/push/PushMessagingService.java]: Do we need to override this method if the only thing we do is call super?

**Review by fourlastor — APPROVED** [review]
One small non blocker

File: app/src/main/java/com/connfa/push/PushMessagingService.java (hunk 1 above)
Comment (rock3r) [reply app/src/main/java/com/connfa/push/PushMessagingService.java]: Ah no, leftover.

Comment (rock3r) [reply app/src/main/java/com/connfa/push/PushMessagingService.java]: ac51672cde933153cdf16c5e56753a2b3cfd853e

**Pull request merged**